steps/s3-cache-pull/0.1.0/step.yml

```diff
@@ -0,0 +1,104 @@
+title: S3 Cache Pull
+summary: |
+  Download your cache from a S3 bucket using custom keys with fallback.
+website: https://github.com/alephao/bitrise-step-s3-cache-pull
+source_code_url: https://github.com/alephao/bitrise-step-s3-cache-pull
+support_url: https://github.com/alephao/bitrise-step-s3-cache-pull/issues
+published_at: 2021-06-01T13:22:50.350941-03:00
+source:
+  git: https://github.com/alephao/bitrise-step-s3-cache-pull.git
+  commit: 98a2b4001302f3d1e9309f57cdb788fb3b3d19b4
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/alephao/bitrise-step-s3-cache-pull
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- aws_access_key_id: $AWS_ACCESS_KEY_ID
```

Comment:
Please don't use default env var values when it is not expected to be available. (exported by other steps as outputs or prefilled by bitrise webhook handling). This applies to the rest of the inputs too.